bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Text.java

```diff
@@ -976,7 +976,7 @@ void fixAlignment () {
 	* style bits instead.
 	*/
 //	if ((style & SWT.BORDER) != 0 && (style & SWT.FLAT) != 0) {
-//		return OS.GetSystemMetrics (OS.SM_CXBORDER);
```

Comment:
Since when has this code been there and commented out?
Is it safe to remove or does someone count on it? (check the git logs for clues)

Reply:
The logs says it has been last change in 2009. Should I remove it or keep it as is?

Reply:
Remove it